packages/cli/src/lib/browser.js

```diff
@@ -1,12 +1,28 @@
 const puppeteer = require('puppeteer');
 const { Renderer } = require('./renderer');
 
-module.exports = async (url) => {  
-  const browser = await puppeteer.launch({
-    headless: true,
-    args: ['--no-sandbox']
-  });
-  const renderer = new Renderer(browser);
-
-  return await renderer.serialize(url);
-};
\ No newline at end of file
+class BrowserRunner {
+
+  constructor() {
+    this.browser = '', this.renderer = '';
```

Comment:
minor, but let's put each variable declaration on its own line.  Also, I think using a `''` is misleading.  Probably just better left `undefined`  or maybe `{}`

Also, what would your thoughts be on folding renderer right in browser?  Doesn't seem all that useful keeping them apart and then you don't have to pass down instances of `browser`?  Just a thought.

Reply:
Yea just initializing, good/bad habit.

Reply:
good habit for sure, I think setting it as an empty string may just be a little misleading is all?  

That said, not sure what closest primitive of an instance of `browser` or `renderer` would be, so 🤷‍♂

Reply:
maybe `null`?  idk, not a huge deal